visualization/app/codeCharta/state/store/metricData/metricData.service.spec.ts

```diff
@@ -0,0 +1,97 @@
+import "../../state.module"
+import { IRootScopeService } from "angular"
+import { MetricDataService } from "./metricData.service"
+import { EdgeMetricDataService } from "./edgeMetricData/edgeMetricData.service"
+import { setEdgeMetricData } from "./edgeMetricData/edgeMetricData.actions"
+import { setNodeMetricData } from "./nodeMetricData/nodeMetricData.actions"
+import { getService, instantiateModule } from "../../../../../mocks/ng.mockhelper"
+import { EDGE_METRIC_DATA, METRIC_DATA, VALID_EDGE, withMockedEventMethods } from "../../../util/dataMocks"
+import { NodeMetricDataService } from "./nodeMetricData/nodeMetricData.service"
+import { StoreService } from "../../store.service"
+import { setEdges } from "../fileSettings/edges/edges.actions"
+
+describe("MetricDataService", () => {
+	let metricDataService: MetricDataService
+	let storeService: StoreService
+	let $rootScope: IRootScopeService
+
+	beforeEach(() => {
+		restartSystem()
+		rebuildService()
+		withMockedEventMethods($rootScope)
+	})
+
+	function restartSystem() {
```

Comment:
```suggestion
	function resetState() {
```

Reply:
`restartSystem` sets up the angular module and the services that need to be injected.
We sometimes put logic into this function to set the state before running the test suite. However, that's not always the case.
And if we rename it here, we'd have to rename it all our test classes increasing the amount of files changed here.

Reply:
I personally recommend to rename this at some point while working on something else in a separate commit or in a separate PR while working on something. It's fine for now.